### DISCUSSION TITLE
新活动低耗刷，刷的时间长会导致重启

### DISCUSSION THREAD

**Discussion by Paradox-Phenomenon** [LmeSzinc/AzurLaneAutoScript#3587] [General]
如题

**Comment by huaiqiu123** [comment]
好像是因为超出了3分钟被alas当作超时了![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/146845569/d0802ee7-278d-4068-8eb4-e7bd4af3a46b)

**Comment by guoh064** [comment]
坐看云起i的低耗点评，别再用5油A胜啦
https://t.bilibili.com/913968310470574098
